cmake/verify-toolchain.cmake

```diff
@@ -16,7 +16,7 @@
 # FORMAT=json: Print the output as a json formatted string, useful for Python
 
 # This is the minimum required Zephyr-SDK version which supports CMake package
-set(TOOLCHAIN_ZEPHYR_MINIMUM_REQUIRED_VERSION 0.13)
+set(TOOLCHAIN_ZEPHYR_MINIMUM_REQUIRED_VERSION 0.13.1)
```

Comment:
We have earlier limited the requested Zephyr SDK based on arch when the SDK was a minor bump affecting only some architectures like here:
https://github.com/zephyrproject-rtos/zephyr/blob/835ff89c405365f5ab332b849bd06473a29f7320/cmake/verify-toolchain.cmake#L19-L23

but as Zephyr `0.13.1` affects both xtensa and arm i'm also fine with just bumping for everyone.

Reply:
so just placing this comment to get general opinions from everyone.